pallets/tx-pause/src/lib.rs

```diff
@@ -64,15 +76,23 @@ pub mod pallet {
         CannotPause,
         /// invalid character encoding
         InvalidCharacter,
+        /// can not resume
+        CannotResume,
```

Comment:
this shouldnt ever happen, right?

Reply:
right, It's in some of my dev iteration, forgot to delete.